docs/DeveloperGuide.md

```diff
@@ -607,8 +607,22 @@ testers are expected to do more *exploratory* testing.
 
 ### Saving data
 
-1. Dealing with missing/corrupted data files
+1. Dealing with missing files
 
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+   1. Delete `config.json` and re-launch the app. Expected: New `config.json` created. Exisiting data is not affected.
```

Comment:
Typo is still there haha, there's still an extra i in your "Existing"

Reply:
gosh so sorry